website/content/docs/networking/cni.mdx

```diff
@@ -204,29 +145,36 @@ You can use the following command to list the iptables rules in the NAT table.
 $ sudo iptables -t nat -L
 ```
 
-## Create your own
-
-You can use this template as a basis for your own CNI-based bridge network
-configuration in cases where you need access to an unsupported option in the
-default configuration, like hairpin mode. When making your own bridge network
-based on this template, ensure that you change the `iptablesAdminChainName` to
-a unique value for your configuration.
```

Comment:
Did this get moved somewhere else in the docs or removed entirely?

Reply:
I moved this to the _Create a CNI brdige mode configuration_ section.